conference.js

```diff
@@ -79,8 +79,6 @@ let initialAudioMutedState = false, initialVideoMutedState = false;
  */
 let DSExternalInstallationInProgress = false;
 
-import {VIDEO_CONTAINER_TYPE} from "./modules/UI/videolayout/VideoContainer";
```

Comment:
The check for VIDEO_CONTAINER_TYPE may not be valid because desktop containers should also be pinned. As such, a check for sharedVideo was added instead.